recipes/benchmark/all/conanfile.py

```diff
@@ -51,7 +51,12 @@ def _configure_cmake(self):
 
         # See https://github.com/google/benchmark/pull/638 for Windows 32 build explanation
         if self.settings.os != "Windows":
-            cmake.definitions["BENCHMARK_BUILD_32_BITS"] = "ON" if "64" not in str(self.settings.arch) else "OFF"
+            if tools.cross_building(self.settings):
+                cmake.definitions["HAVE_STD_REGEX"] = False
```

Comment:
interesting, isn't it something that is usually auto-detecting? e.g. by [CheckCXXSourceCompilers](https://cmake.org/cmake/help/latest/module/CheckCXXSourceCompiles.html)

Reply:
ping @uilianries

Reply:
@SSE4 indeed, but they don't use this cmake method in benchmark and it's a known limitation: https://github.com/google/benchmark/issues/351